### ISSUE TITLE
bug: When closing drawer in responsive dialog example elements on the screen are not longer interactable

### ISSUE THREAD

**Issue by ieedan** [huntabyte/shadcn-svelte#1549]
### Describe the bug

Opening this here because I'm' not sure where exactly it's going to belong. 

When you resize the screen while the responsive dialog is open so that you go from using the dialog to the drawer, once you close the drawer any elements on the screen will not be clickable. It's almost like there's still an invisible overlay.

This doesn't happen when just using the drawer component alone since there is no transition between the two.

https://github.com/user-attachments/assets/f26a3724-e6e4-4bf9-9a11-ec7fc33f277c

### Reproduction

https://next.shadcn-svelte.com/docs/components/drawer

### Logs

```bash
N/A
```

### System Info

```bash
N/A
```

### Severity

annoyance

**Comment by ieedan** [comment]
> Well, proud to say that this issue isn't the case for Firefox. 😂

Must be a chromium thing then...

**Comment by ieedan** [comment]
Yes in my other projects using this they are updated to `1.0.0-next.3` and `1.0.0-next.71`.

**Comment by thewengchan** [comment]
Found out the cause for this behaviour after a few debugging. It's in /node_modules/bits-ui/dist/internal/body-scroll-lock.svelte.js

Apparently it locks the body:

`
afterTick(() => {
            document.body.style.pointerEvents = "none";
            document.body.style.overflow = "hidden";
        });
`
